source/educators/navigation/components_activities.rst

```diff
@@ -96,7 +96,6 @@ Drag and Drop Problem
    :maxdepth: 2
    :glob:   
 
-   ../references/course_development/exercise_tools/drag_and_drop.rst
```

Comment:
I think you want to keep this in, and delete L104 (the file you deleted)

Reply:
Yes, but still the error is happening.